autoload/ctrlsf/win.vim

```diff
@@ -156,6 +156,11 @@ func! ctrlsf#win#CloseMainWindow() abort
       call ctrlsf#win#RestoreAllWinSize()
 
       call ctrlsf#win#FocusCallerWindow()
+
+      " hook for user customization
+      if exists("*g:CtrlSFAfterMainWindowClose")
+        silent! call g:CtrlSFAfterMainWindowClose()
```

Comment:
As ctrlsf window is closed, this method will be called on caller window, which should be avoided.